cmd/vela-worker/flags.go

```diff
@@ -85,6 +85,11 @@ func flags() []cli.Flag {
 			Name:    "server.cert-key",
 			Usage:   "optional TLS certificate key",
 		},
+		&cli.StringFlag{
+			EnvVars: []string{"WORKER_SERVER_TLS_MIN_VERSION", "VELA_SERVER_TLS_MIN_VERSION", "SERVER_TLS_MIN_VERSION"},
+			Name:    "server.tls-min-version",
+			Usage:   "optional TLS minimum version requirement",
+		},
```

Comment:
any reason to not set the default here where we define the flag like we do with other values?